util/testutil/testutil.go

```diff
@@ -91,3 +92,18 @@ func (checker *datumEqualsChecker) Check(params []interface{}, names []string) (
 	}
 	return res == 0, ""
 }
+
+// RowsWithSep is a convenient function to wrap args to a slice of []interface.
+// The arg represents a row, split by sep.
+func RowsWithSep(sep string, args ...string) [][]interface{} {
```

Comment:
This is a function with the same name and the same behavior in util/testkit/testkit.go.